gallery/src/components/NxVulnerabilityCounter/NxVulnerabilityCounterPage.tsx

```diff
@@ -0,0 +1,176 @@
+/*
+ * Copyright (c) 2019-present Sonatype, Inc.
+ * This program and the accompanying materials are made available under
+ * the terms of the Eclipse Public License 2.0 which accompanies this
+ * distribution and is available at https://www.eclipse.org/legal/epl-2.0/.
+ */
+import React from 'react';
+import { NxTextLink, NxTable, NxP, NxCode } from '@sonatype/react-shared-components';
+
+import { GalleryDescriptionTile, GalleryExampleTile } from '../../gallery-components/GalleryTiles';
+
+import NxVulnerabilityCounterExample from './NxVulnerabilityCounterExample';
+import NxVulnerabilityCounterWrappingExample from './NxVulnerabilityCounterWrappingExample';
+import NxVulnerabilityCounterSmallExample from './NxVulnerabilityCounterSmallExample';
+import NxVulnerabilityCounterColumnExample from './NxVulnerabilityCounterColumnExample';
+import NxVulnerabilityCounterSmallColumnExample from './NxVulnerabilityCounterSmallColumnExample';
+import NxVulnerabilityCounterGridExample from './NxVulnerabilityCounterGridExample';
+import NxVulnerabilityCounterSmallGridExample from './NxVulnerabilityCounterSmallGridExample';
+
+const NxVulnerabilityCounterCode = require('./NxVulnerabilityCounterExample?raw'),
+    NxVulnerabilityCounterWrappingCode = require('./NxVulnerabilityCounterWrappingExample?raw'),
+    NxVulnerabilityCounterSmallCode = require('./NxVulnerabilityCounterSmallExample?raw'),
+    NxVulnerabilityCounterColumnCode = require('./NxVulnerabilityCounterColumnExample?raw'),
+    NxVulnerabilityCounterSmallColumnCode = require('./NxVulnerabilityCounterSmallColumnExample?raw'),
+    NxVulnerabilityCounterGridCode = require('./NxVulnerabilityCounterGridExample?raw'),
+    NxVulnerabilityCounterSmallGridCode = require('./NxVulnerabilityCounterSmallGridExample?raw');
+
+const NxVulnerabilityCounterPage = () =>
+  <>
+    <GalleryDescriptionTile>
+      <NxP>
+        Component for a series of small counters which display a number representing a
```

Comment:
Like my comment on the other page, I think it should be spelled out that the grouping here is done by the CVSS rating of the CVSS scores of the vulnerabilities being counted.

Reply:
Applied your suggestion here: [c9496b9](https://github.com/sonatype/sonatype-react-shared-components/pull/1200/commits/c9496b9e4ae232c1b9c3b91b89916498716fe9a6)